Neos.Neos/Classes/Domain/Repository/UserRepository.php

```diff
@@ -24,6 +25,9 @@
  */
 class UserRepository extends Repository
 {
+    const SORT_BY_LASTLOGGEDIN = 'LastLoggedIn';
+    const SORT_DIRECTION_DESC = 'DESC';
```

Comment:
This constant is already defined as `\Neos\Flow\Persistence\QueryInterface::ORDER_DESCENDING` - would make sense to use that one.